downstream/modules/platform/proc-controller-add-source.adoc

```diff
@@ -0,0 +1,116 @@
+* *Source*: Choose a source for your inventory. 
+For more information on sources, and supplying the appropriate information, see [Inventory sources].
+
+. When the information for your chosen [Inventory sources] is complete, you can optionally specify other common parameters, such as verbosity, host filters, and variables.
+
+. Use the *Verbosity* menu to select the level of output on any inventory source's update jobs.
+. Use the *Host Filter* field to specify only matching host names to be imported into {ControllerName}.
+. In the *Enabled Variable* field, specify that {ControllerName} retrieves the enabled state from the dictionary of host variables. 
+You can specify the enabled variable using dot notation as 'foo.bar', in which case the lookup searches nested dictionaries, equivalent to: `from_dict.get('foo', {}).get('bar', default)`.
+. If you specified a dictionary of host variables in the *Enabled Variable* field, you can provide a value to enable on import. 
+For example, if `enabled_var='status.power_state'` and `enabled_value='powered_on'` in the following host variables, the host is marked `enabled`:
++
+[literal, options="nowrap" subs="+attributes"]
+----
+{
+"status": {
+"power_state": "powered_on",
+"created": "2020-08-04T18:13:04+00:00",
+"healthy": true
+},
+"name": "foobar",
+"ip_address": "192.168.2.1"
+}
+----
++
+If `power_state` is any value other than `powered_on`, then the host is disabled when imported into {ControllerName}. 
+If the key is not found, then the host is enabled.
+
+. All cloud inventory sources have the following update options:
+
+* *Overwrite*: If checked, any hosts and groups that were previously present on the external source but are now removed, are removed from
+the {ControllerName} inventory. 
+Hosts and groups that were not managed by the inventory source are promoted to the next manually created group, or, if there is no manually created group to promote them into, they are left in the "all" default group for the inventory.
++
+When not checked, local child hosts and groups not found on the external source remain untouched by the inventory update process.
+* *Overwrite Variables*: If checked, all variables for child groups and hosts are removed and replaced by those found on the external source. 
++
+When not checked, a merge is performed, combining local variables with those found on the external source.
+* *Update on Launch*: Each time a job runs using this inventory, refresh the inventory from the selected source before executing job tasks. 
++
+To avoid job overflows if jobs are spawned faster than the inventory can synchronize, selecting this enables you to configure a *Cache Timeout* to previous cache inventory synchrinizations for a certain number of seconds.
```

Comment:
```suggestion
To avoid job overflows if jobs are spawned faster than the inventory can synchronize, selecting this enables you to configure a *Cache Timeout* to previous cache inventory synchronizations for a certain number of seconds.
```